src/parser.rs

```diff
@@ -274,7 +274,6 @@ pub enum MasterPlaylistTag {
     SessionKey(SessionKey),
     Start(Start),
     IndependentSegments,
-    Unknown(ExtTag),
```

Comment:
Are you sure? Because MasterPlaylists are parsed entirely seperately from MediaPlaylist/MediaSegment, it really shouldn't be affected by these changes unless i'm mistaken.